pytorch_toolkit/formula_recognition/README.md

```diff
@@ -0,0 +1,199 @@
+# PyTorch realization of the Formula Recognition
+
+This code is based on a [PyTorch realization](https://github.com/luopeixiang/im2latex/) of the code from the original [repository](https://github.com/harvardnlp/im2markup/).
+Models code is designed to enable ONNX\* export and inference on CPU\GPU via OpenVINO™.
+
+## Setup
+
+### Prerequisites
+
+* Ubuntu\* 16.04
+* Python\* 3.7 or newer
+* PyTorch\* (1.4.0)
+* OpenVINO™ 2020.4 with Python API
+
+### Optional prerequisites
+
+#### Install required packages for evaluation
+These packages are used for rendering images while evaluation and demo.
+
+```bash
+sudo apt-get update &&
+  apt-get install -y --no-install-recommends \
+    texlive \
+    imagemagick \
+    ghostscript
+```
+
+### Installation
+
+Create and activate virtual environment:
+
+```bash
+bash init_venv.sh
+```
+
+### Download or Prepare Datasets
+
```

Comment:
Vladimir, if we add a new subsection `#### Vocabulary files` as I wrote in my previous comment, then we should add a new subsection here (otherwise the tree of the document won't be "balanced").

My suggestion is to add a new subsection header at this point as follows:
```
#### Dataset format
```